Google-Summer-of-Code-2023.md

```diff
@@ -0,0 +1,261 @@
+## Table of Contents
+
+- [Getting started](#getting-started)
+- [FAQs](#faqs)
+- [Dates and Deadlines](#dates-and-deadlines)
+- [Types of work related to Oppia projects](#types-of-work-related-to-oppia-projects)
+- [GSoC proposal template](#gsoc-proposal-template)
+  - [Tips for writing a good project plan](#tips-for-writing-a-good-project-plan)
+  - [What should applicants expect from mentors in a proposal review?](#what-should-applicants-expect-from-mentors-in-a-proposal-review)
+  - [Sample proposals from past years](#sample-proposals-from-past-years)
+- [Selection Criteria](#selection-criteria)
+- [Communication](#communication)
+- [Oppia's Project Ideas List](#oppias-project-ideas-list)
+
+This year, Oppia will be participating in [Google Summer of Code 2023](https://summerofcode.withgoogle.com/)! GSoC is a global program which offers non-experienced contributors (and, historically, post-secondary students) the opportunity to discover and work with open-source organizations. The contributions are supported by a stipend. Contributors work closely with one or more mentors from an open-source organization to implement either a project idea by the organization, or a proposal of their own. You might be interested in our GSoC info pages from previous years: [2022](https://github.com/oppia/oppia/wiki/Google-Summer-of-Code-2022), [2021](https://github.com/oppia/oppia/wiki/Google-Summer-of-Code-2021), [2020](https://github.com/oppia/oppia/wiki/Google-Summer-of-Code-2020), [2019](https://github.com/oppia/oppia/wiki/Google-Summer-of-Code-2019), [2018](https://github.com/oppia/oppia/wiki/Google-Summer-of-Code-2018), [2017](https://github.com/oppia/oppia/wiki/Google-Summer-of-Code-2017), [2016](https://github.com/oppia/oppia/wiki/Google-Summer-of-Code-2016).
```

Comment:
We can change the second sentence of this, since students are explicitly allowed in 2023. See https://developers.google.com/open-source/gsoc/faq -- "GSoC is for new and beginner contributors to open source and to students".

Maybe we can write: "which offers students and non-experienced contributors the opportunity..."